docs/project_defn.rst

```diff
@@ -283,7 +283,10 @@ on the `GCP Batch <https://cloud.google.com/batch>`_ service.
     *  ``prefix``: The Cloud Storage prefix at which the data will be stored.
 
 *  ``region``: The GCP region in which the batch will be run and of the Artifact Registry.
-*  ``batch_array_size``: Number of concurrent simulations to run. Max: 10000.
+*  ``batch_array_size``: Number of tasks to divide the simulations into. Max: 10000.
+*  ``parallelism``: Optional. Maximum number of tasks that can run in parallel. If not specified,
+   uses `GCP's default behavior`_ ("Default to ``min(task_count, parallel tasks per job limit)``").
```

Comment:
Optional: Might be more useful to link to the page with the actual limits, rather than the slightly mysterious "tasks per job limit".

```suggestion
   uses `GCP's default behavior`_ (parallelism equals batch_array_size, up to the `job limits`_).
 
 .. _job limits: https://cloud.google.com/batch/quotas
```